lib/APIDigester/ModuleAnalyzerNodes.cpp

```diff
@@ -1742,6 +1742,7 @@ SDKContext::shouldIgnore(Decl *D, const Decl* Parent) const {
     case AccessLevel::Private:
     case AccessLevel::FilePrivate:
       return true;
+    case AccessLevel::Package:
     case AccessLevel::Public:
     case AccessLevel::Open:
       break;
```

Comment:
We may want to revisit this later. I believe this logic is used by the ABI checker to report changes that would break clients that were built against an old ABI and couldn't use the new ABI at runtime. Package clients should always be version locked, the ABI checker can likely ignore package ABI breaks.

Reply:
As is it's more restrictive, so making it more permissive later shouldn't be a problem.

Reply:
Good to know. Will add a note to keep track.